centreon-gorgone/packaging/centreon-gorgone-selinux.yaml

```diff
@@ -24,6 +24,13 @@ conflicts:
 provides:
   - centreon-gorgone-selinux-debuginfo
 
+overrides:
```

Comment:
since this package is only built to rpm, you can directly move this to root `replaces` and `provides` (NOT depends)